### ISSUE TITLE
App detail pages are not accessible to screen reader

### ISSUE THREAD

**Issue by cassidyjames** [ubuntu/app-center#1587]
When I open an app detail page with Ubuntu's built-in screen reader, I don't get _any_ information about the contents of the page. Further, I cannot focus most of the text on the screen with the keyboard, meaning I cannot explore with the keyboard to have it read out.
